feature-libs/cart/saved-cart/styles/_saved-cart-list.scss

```diff
@@ -129,6 +129,11 @@ cx-saved-cart-list {
     .cx-saved-cart-list-label {
       color: var(--cx-color-secondary);
 
+      // TODO: (CXSPA-5955) Remove feature flag next major
+      @include forVersion(6.8) {
```

Comment:
Styling changes here will need to follow the new [style versioning method](https://github.com/SAP/spartacus/pull/18523).

Reply:
Changed for the new method.